charts/opencloud/README.md

```diff
@@ -481,125 +481,7 @@ Apply the certificate:
 kubectl apply -f cluster-issuer.yaml
 ```
 
-### Step 4: Create the Gateway
-
-Create a Gateway resource to expose your services:
-
-
-```yaml
-# gateway.yaml
-apiVersion: gateway.networking.k8s.io/v1beta1
-kind: Gateway
-metadata:
-  name: cilium-gateway
-  namespace: kube-system
-spec:
-  gatewayClassName: cilium
-  infrastructure:
-    annotations:
-      io.cilium/lb-ipam-ips: "192.168.178.77"  # Replace with your desired IP
-      cilium.io/hubble-visibility: "flow"
-      cilium.io/preserve-client-cookies: "true"
-      cilium.io/preserve-csrf-token: "true"
-      io.cilium/websocket: "true"
-      io.cilium/websocket-timeout: "3600"
```

Comment:
You are dropping all of these in the below `helm install opencloud` cli. That might create problems.